javascript-data-model/order-history.js

```diff
@@ -0,0 +1,55 @@
+var orderHistory = [
+  {
+    orderNumber: '114-3941689-8772232',
+    datePlaced: 'August 4, 2020',
+    dateDelivered: 'August 8, 2020',
+    totalPrice: '$34.00',
+    shippedTo: 'JS Masher',
+    book: {
+      author: 'Dr. Axel Raushmayer',
+      title: 'JavaScript for impatient programmers',
+      price: '$31.55'
+    }
+  },
+  {
+    orderNumber: '113-9984268-1280257',
+    datePlaced: 'July 19, 2020',
+    dateDelivered: 'July 20, 2020',
+    totalPrice: '$44.53',
+    shippedTo: 'JS Masher',
+    book: {
+      author: 'Christopher Alexander',
+      title: 'The Timeless Way of Building',
+      price: '$41.33'
+    }
+  },
+  {
+    orderNumber: '114-2875557-9059409',
+    datePlaced: 'July 4, 2020',
+    dateDelivered: 'July 7, 2020',
+    totalPrice: '$17.22',
+    shippedTo: 'JS Masher',
+    gameEquipment: {
+      console: 'GameCube',
+      equipmentType: 'Controller Adapter',
+      price: '$15.98'
+    }
+  },
```

Comment:
In one of these orders, there is a `book` property, but in the other, there is a `gameEquipment` property. This formatting is inconsistent and would lead to bugs if somebody tried to use this data.